packages/backend/src/controllers/authentication.ts

```diff
@@ -150,7 +150,11 @@ const createOpenIdUserFromProfile = (
 
 const setupClient = async () => {
     const oktaIssuer = await Issuer.discover(
-        `https://${lightdashConfig.auth.okta.oktaDomain}/oauth2/default`,
+        `${lightdashConfig.auth.okta.oauth2Issuer}${
+            lightdashConfig.auth.okta.authorizationServerId
+                ? `/oauth2${lightdashConfig.auth.okta.authorizationServerId}`
+                : ''
+        }`,
```

Comment:
used `oauth2Issuer` because it must contain the `https://` and only append `/oauth2` when there's an `authorizationServerId`

Tested this with: 
* `default` authorizationServerId
* no `authorizationServerId`